### DISCUSSION TITLE
Include `run_completed_at` timestamp in `/repos/{owner}/{repo}/actions/runs` response

### DISCUSSION THREAD

**Discussion by abhijit-hota** [community/community#81461] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Currently the [list of workflow runs](https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#list-workflow-runs-for-a-repository) only returns `run_started_at` and not the completion timestamp. 

The timestamp is needed to calculate the total runtime. I am currently relying on the [workflow run usage API](https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#get-workflow-run-usage) but we're easily hitting rate limits doing that for all the required workflow runs separately.

Returning a `run_completed_at` timestamp would be welcome. Also, I'd like to know if there is a way to do this using the GraphQL API.

**Comment by ashkan0201** [comment]
Response #1

**Reply by abhijit-hota** [reply]
Can you clarify what I need to change in my post if at all? I couldn't figure out how to edit the labels if that's what you meant. 

Thanks.

**Reply by abhijit-hota** [reply]
Friendly bump. @ashkan0201
